src/cost_estimates/fee_scalar.rs

```diff
@@ -0,0 +1,207 @@
+use std::cmp;
```

Comment:
Minor Q on code organization -- should `src/cost_estimates` be moved under `src/chainstate/stacks`, since it pertains mainly to processing Stacks chainstate?